juju-charms/charm-trilio-data-mover/src/reactive/trilio_data_mover.py

```diff
@@ -208,34 +202,35 @@ def create_virt_env():
     try:
         # remove old venv if it exists
         os.system('rm -rf {}'.format(venv_path))
-        venv_src = 'http://{}:8081/packages/queens_ubuntu'\
-                   '/tvault-contego-virtenv.tar.gz'.format(tv_ip)
-        venv_dest = path
-        handler.install(venv_src, venv_dest)
+        venv_dest = '/'
+        handler.install(venv_url, venv_dest)
         log("Virtual Environment installed successfully")
     except Exception as e:
         log("Failed to install Virtual Environment")
         return False
 
+    # Install TrilioVault Datamover package
+    if not install_plugin(pkg_name):
+        return False
+
     # Get dependent libraries paths
     try:
-        cmd = ['/usr/bin/python', 'files/trilio/get_pkgs.py']
+        cmd = ['/usr/bin/python{}'.format(config('python-version')),
+               'files/trilio/get_pkgs.py']
         sym_link_paths = check_output(cmd).decode('utf-8').strip().split('\n')
     except Exception as e:
         log("Failed to get the dependent packages--{}".format(e))
         return False
 
-    # Install TrilioVault Datamover package
-    if not install_plugin(tv_ip, latest_dm_ver, '/usr'):
-        return False
-
     # Create symlinks of the dependent libraries
-    venv_pkg_path = '{}/lib/python2.7/site-packages/'.format(venv_path)
+    venv_pkg_path = '{}/lib/python3.6/site-packages/'.format(venv_path)
```

Comment:
Do we need to handle this for py2?

Reply:
Virtual environment will not contain python2 libraries. So we have moved it to python3 only. As it is a virtual env, we can fix the python version.